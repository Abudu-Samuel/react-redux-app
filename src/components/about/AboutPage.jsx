import React from 'react';

const AboutPage = () => (
  <div>
    <h1>About</h1>
    <p>This application uses React, Redux, React router dom and other helpful libraries</p>
  </div>
);

export default AboutPage;