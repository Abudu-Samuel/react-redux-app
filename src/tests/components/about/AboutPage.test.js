/* eslint-disable */
import AboutPage from '../../../components/about/AboutPage';

describe('<AboutPage />', () => {
  it('render routes in the app', () => {
    const wrap = shallow(<AboutPage />);
    expect(toJson(wrap)).toMatchSnapshot();
  });
});