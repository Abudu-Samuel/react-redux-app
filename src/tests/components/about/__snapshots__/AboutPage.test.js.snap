// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutPage /> render routes in the app 1`] = `
<div>
  <h1>
    About
  </h1>
  <p>
    This application uses React, Redux, React router dom and other helpful libraries
  </p>
</div>
`;
